Source/Fuse.PushNotifications/Android/Impl.uno

```diff
@@ -299,6 +302,15 @@ namespace Fuse.PushNotifications
 				.setAutoCancel(true)
 				.setContentIntent(pendingIntent);
 
+
+				if(notificationStyle=="bigtextstyle" || notificationStyle.equals("bigtextstyle")){
```

Comment:
Use a const string.

Reply:
I don't understand. Do you mean that I should create a constant string with the value "bigtextstyle" and then compare it with notificationStyle value?

Reply:
Yes. Anytime a string is used more than once it should be a constant.